idaes/core/util/tests/test_tag_svg.py

```diff
@@ -22,7 +22,8 @@
 from pyomo.environ import ConcreteModel, Set, Block, Var
 from pyomo.network import Port
 
-from idaes.core.util.misc import TagReference, svg_tag
+from idaes.core.util.misc import TagReference
+from idaes.core.util.tags import svg_tag
```

Comment:
If this was done right, the new `svg_tag` should have its own tests so we should be able to remove this instead. If not, then we should move these tests over to the `test_tags.py` file.

Reply:
I don't think svg_tag really has its own test since it is only referenced in "lazy testing" where it confirms the data from the xml files. I just moved everything from this file into test_tags.py . Should I try to write new tests for svg_tag or leave it to someone else?

Reply:
The existing (moved) tests should be sufficient for now.